modules/openapi-generator/src/main/resources/python/pyproject.mustache

```diff
@@ -43,7 +43,7 @@ pydantic = ">= 2"
 typing-extensions = ">= 4.7.1"
 {{/poetry1}}
 {{^poetry1}}
-requires-python = "^3.9"
+requires-python = ">3.8"
```

Comment:
thanks for the PR.

minor suggestion. what about the following instead?

```
requires-python = ">=3.9"
```

to me that's easier to understand 3.9 is the minimal version supported